packages/trpc/server/routers/viewer/bookings/requestReschedule.handler.ts

```diff
@@ -297,6 +297,7 @@ export const requestRescheduleHandler = async ({ ctx, input }: RequestReschedule
     sendPayload(webhook.secret, eventTrigger, new Date().toISOString(), webhook, {
       ...evt,
       smsReminderNumber: bookingToReschedule.smsReminderNumber || undefined,
+      cancelledBy: user.email,
```

Comment:
what user is this? 🤔

Reply:
It would be whatever user is using the Request Reschedule button in the booking tab.

![Screenshot 2024-08-21 191510](https://github.com/user-attachments/assets/e767a4db-ca37-407d-9cdf-0fce46a2d26e)